spring-grpc-spring-boot-autoconfigure/pom.xml

```diff
@@ -84,8 +84,16 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter</artifactId>
```

Comment:
The `provided` did not play well w/ `optional` actuator. I believe just adding in w/o the boot starter works well too.